packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss

```diff
@@ -6,17 +6,24 @@
     max-height: 170px;
     max-width: calc(100% - 50px});
     overflow: hidden;
-
+    text-align: center;
     @media screen and (max-width: $terra-medium-breakpoint) {
       margin: 40px 25px;
     }
   }
 
   .img {
-    color: #3d3d3d;
-    display: block;
+    color: var(--terra-consumer-logo-text-color, #3d3d3d);
+    display: inline-block;
     margin: auto;
     max-height: 120px;
     max-width: 100%;
   }
+
+  .logo-text {
+    color: var(--terra-consumer-logo-text-color, #3d3d3d);
+    font-size: 30px;
+    font-weight: normal;
+    text-shadow: 0.25px 0.25px;
```

Comment:
wouldn't setting these on the .img class be applied to the alternate text anyway? why make the second class?

Reply:
Anna said we don't want text shadow for now so removed it.